masterlist.yaml

```diff
@@ -591,26 +598,28 @@ plugins:
       - <<: *corrupt
         condition: 'checksum("RealisticDamageMultipliers.esm", CBC568FF)'
 
-###### Gameplay - Crafting & Harvesting ######
+  ###### Gameplay - Crafting & Harvesting ######
   - name: 'legendaries\.es[mp]'
     url:
       - link: 'https://www.nexusmods.com/starfield/mods/1379/'
         name: 'Legendary Modifications'
   - name: 'legendaries.esp'
     msg:
       - <<: *obsolete
-        subs: [ '[Legendary Modifications .esm Plugin](https://www.nexusmods.com/starfield/mods/1379/)' ]
+        subs:
+          [
+            '[Legendary Modifications .esm Plugin](https://www.nexusmods.com/starfield/mods/1379/)',
+          ]
```

Comment:
How come this gets split over multiple lines, does the formatter have a soft line length limit?

Reply:
I fixed this here: https://github.com/loot/starfield/pull/74/commits/a537b915d18bb620f6965f5c9dafb2ba36f8acc0